automation/common.sh

```diff
@@ -3,6 +3,16 @@
 # Common functions for the scripts
 #
 
+
+set_guestfs_params() {
+    # see: https://bugzilla.redhat.com/show_bug.cgi?id=1404287
+    export LIBGUESTFS_APPEND="edd=off"
+    # make libguestfs use /dev/shm as tmpdir
+    export LIBGUESTFS_CACHEDIR="/dev/shm"
```

Comment:
No significant improvements :/ trimmed around 30 seconds from the functional tests here(of check-path) to ~490 seconds.
I suggest we keep it like this, as it probably can't harm.